packages/@sanity/components/src/previews/DefaultPreview.js

```diff
@@ -54,13 +47,16 @@ class DefaultPreview extends React.PureComponent {
 
     if (isPlaceholder) {
       return (
-        <div className={media ? styles.hasMedia : ''}>
-          {PLACEHOLDER}
```

Comment:
Did inlining the placeholder fix the Firefox bug?

Reply:
no. I replaced it totally with only html/css instead of SVG. We looses the gradient styling that SVG can do, but this is more robust. The other previews needs the same when we turn them on.